README.md

```diff
@@ -1,16 +1,18 @@
+# get-changed-files
```

Comment:
I did a bit of linting on the README too, I hope that's ok

Reply:
That's awesome, thanks!